src/interfaces/IRfqModule.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.13;
+
+import {IBaseModule} from "./IBaseModule.sol";
+import {IPerpAsset} from "v2-core/src/interfaces/IPerpAsset.sol";
+import {IAsset} from "v2-core/src/interfaces/IAsset.sol";
+
+interface IRfqModule is IBaseModule {
+  struct RfqOrder {
+    uint maxFee;
+    TradeData[] trades;
+  }
+
+  struct TakerOrder {
+    /// @dev a hash of the RfqOrder being traded against
+    bytes32 orderHash;
+    uint maxFee;
+  }
+
+  struct TradeData {
+    address asset;
+    uint subId;
+    int markPrice;
+    int amount;
+  }
+
+  struct OrderData {
+    uint makerAccount;
+    uint makerFee;
+    uint takerAccount;
+    uint takerFee;
+    bytes managerData;
```

Comment:
maybe we should stop called them managerData here cuz it's no longer passed onto the manager

Reply:
eh, I cant think of a better word here, seems fine since it's packed the same/handled the same as managerData